src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1592409240.inc.php

```diff
@@ -0,0 +1,26 @@
+<h1>Build #1592409240</h1>
+<h2>Date: 2020-06-17</h2>
+<div class="changelog">
+    - ref #595 add missing target table config to document tree parent field
+</div>
+<?php
+$documentTreeParentFieldId = TCMSLogChange::GetTableFieldId(
+    TCMSLogChange::GetTableId('cms_document_tree'),
+    'parent_id'
+);
+$query = "SELECT `fieldtype_config` FROM `cms_field_conf` WHERE `id` = :id";
+$fieldConfig = TCMSLogChange::getDatabaseConnection()->fetchColumn($query, ['id' => $documentTreeParentFieldId]);
+if (false === stripos($fieldConfig, 'connectedTableName')) {
+    $data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'de')
+        ->setFields(
+            [
+                'fieldtype_config' => 'connectedTableName=cms_document_tree',
```

Comment:
Could there be another fieldtype_config already?

Reply:
you mean one that configures everything but the connected table (because I check that in line 13). I think that is very very unlikely seeing as the field is broken as long as the connected table is not configured. So I would argue, that if someone added a configuration, then they would have had to also set the connected table name in order for the field to work.